dali/core/mm/cu_vm_test.cc

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/core/mm/cu_vm.h"
+#include <gtest/gtest.h>
+
+#if DALI_USE_CUDA_VM_MAP
+
+namespace dali {
+namespace mm {
+namespace test {
+
+TEST(CUMemAddressRange, Reserve) {
+  ASSERT_TRUE(cuInitChecked());
```

Comment:
Do you need to call it? Wouldn't other functions do that?

Reply:
Well, I can remove it to check that the functions actually load - but it wouldn't matter unless running a single test.